build.gradle

```diff
@@ -199,7 +199,9 @@ dependencies {
     implementation 'org.apache.directory.studio:org.apache.commons.io:2.4'
     implementation 'org.liquibase:liquibase-core:3.8.0'         // For upgrade
     implementation 'org.postgresql:postgresql:42.2.18'          // Postgres jdbc driver
-    implementation 'org.slf4j:slf4j-api:1.7.28'                 // Logging facade
+    implementation 'ch.qos.logback:logback-core:1.2.13'
```

Comment:
Would you mind adding a comment that links to the Spring Boot upgrade ticket so that we’re more likely to revisit them when the time comes?